pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+### Knowledge Base
+<!--
+_If the PR introduces new concept, then provide link(s) to the Knowledge base ( Documentation, Blog etc )_
+-->
+
+### Example(s)
+<!--
+_Any details that will help reviewers to review the PR, such as  (but not limited to ) expected request/response, instruction for viewing the changes via the web client, or any other relevant information._
+-->
+
+### Checklist
+
+- [ ] Black formatting
```

Comment:
This is specific to Python so not for all PRs but good to keep as reminder in template. We could make it more generic ("linting tools") or add more examples (e.g. do we also want type checking).

Reply:
Why do you want linting/type checking to be a manual step; why not make it a part of CI/CD pipeline?